bindings/pyroot/cppyy/CPyCppyy/src/Pythonize.cxx

```diff
@@ -841,6 +841,16 @@ PyObject* name##StringCompare(PyObject* self, PyObject* obj)                 \
 CPPYY_IMPL_STRING_PYTHONIZATION_CMP(std::string, Stl)
 CPPYY_IMPL_STRING_PYTHONIZATION_CMP(std::wstring, StlW)
 
+Py_hash_t StlStringHash(PyObject* self)
+{
+// std::string objects hash to the same values as Python strings to allow
+// matches in dictionaries etc.
+    PyObject* data = StlStringGetData(self);
```

Comment:
Does `StlStringGetData` has a reference?

Reply:
Looks like it's the caller who needs to decref it, see [this](https://github.com/root-project/root/blob/c92cf8e9601fe0a4b37ac046edaf158d9013db40/bindings/pyroot/cppyy/CPyCppyy/src/Pythonize.cxx#L782-L785).